src/index.html

```diff
@@ -1,13 +1,77 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div data-qa="card" class="card">
+
+      <div class="card__image-container">
+
+        <img
+          src="images/imac.jpeg"
+          alt="Imac Img"
+          class="card__image"
+        >
+      </div>
+
+      <h1 class="card__heading">
+
+        APPLE A1419 iMac 27" Retina
+        <br>
+
+        5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <h2 class="card__code">
+
+        Product code: 195434
+      </h2>
```

Comment:
1. it's better to use p tag
```suggestion
      <p class="card__code">
        Product code: 195434
      </p>
```
2. remove extra empty lines